src/main/scala-sbt-0.13/com/typesafe/sbteclipse/core/Eclipse.scala

```diff
@@ -305,11 +305,22 @@ private object Eclipse extends EclipseSDTConfig {
       srcEntries <- srcEntriesIoSeq.toList.sequence;
       linkEntries <- srcLinkEntriesIoSeq.toList.sequence
     ) yield {
+      val eodirs: Seq[String] = for {
```

Comment:
Could you rename these variables to `eclipseOutputDirs` and `ecilpseOutputPath`? I'm not sure most people would realize what `eo` stands for

Reply:
pushed these edits a few minutes ago.